src/components/Filters/components/TagsWrapper/tests/TagsWrapper.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {mountWithApp} from 'test-utilities';
+
+import {TagsWrapper} from '../TagsWrapper';
+import {VisuallyHidden} from '../../../../VisuallyHidden';
+
+const MockChild = () => <div />;
+
+describe('<TagsWrapper />', () => {
+  it('renders visually hidden component when shouldHide is true', () => {
+    const tagsWrapper = mountWithApp(
+      <TagsWrapper shouldHide>
+        <MockChild />
+      </TagsWrapper>,
+    );
+
+    expect(tagsWrapper).toContainReactComponentTimes(VisuallyHidden, 1);
+    expect(tagsWrapper).toContainReactComponentTimes(MockChild, 1);
```

Comment:
Is it worth asserting that MockChild is inside the VisuallyHidden? Something like:

```suggestion
    expect(tagsWrapper.find(VisuallyHidden)).toContainReactComponent(MockChild);
```